plugins/techdocs/api-report.md

```diff
@@ -133,7 +135,9 @@ export type DocsTableRow = {
 
 // @public
 export const EmbeddedDocsRouter: (
-  props: PropsWithChildren<{}>,
+  props: PropsWithChildren<{
+    emptyState?: React_2.ReactElement;
```

Comment:
We need to avoid introducing this in the old system and external API. I don't think this overall solution is where we're going to end up with the API in the end as it requires a lot of drilling, and TechDocs is 1.0